SCAN.md

```diff
@@ -1,5 +1,20 @@
 Documentation on the Seattle Coronavirus Assessment Network (SCAN) project
 
+- [Adding a new study arm](#adding-a-new-study-arm)
+- [Adding a new language](#adding-a-new-language)
+
+## Adding a new study arm
+
+1. First, answer the questions listed in the [barcodes documentation] pertaining to creating new collections.
+2. Create a new identifier set.
+3. Create a new barcode label layout for the [labelmaker].
+4. Enable REDCap DETs for the new project once testing is complete and it's ready for production.
+5. Update the REDCap DET ETL with the new study's project ID and purview (see [example]).
+6. Add a new cronjob for the new REDCap DET ETL.
+7. ((placeholder for SCAN RoR workflow changes))
+8. Update the [export-record-barcodes] script in the [scan-switchboard] repo with the new language's project ID and purview. ((?))
```

Comment:
Is the Switchboard for unboxing mail kits only, or would the lab like to look up any SCAN barcode?